.circleci/config.yml

```diff
@@ -57,6 +57,11 @@ jobs:
       - run: bundle exec rake db:create
       - run: bundle exec rake db:schema:load
 
+      - run:
+          name: run diff on schema.json
+          command: |
+            test/schema_diff.sh
```

Comment:
I wonder if we could fit this onto one line, instead of adding a script file? What do you think about this?
```sh
command: git diff --quiet schema.json || exit 1
```

Reply:
or maybe this? https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code

Reply:
Sounds good, this is a lot simpler!